src/components/Contact.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useEffect, useState } from 'react'
+import './Contact.css'
+import ValidateForm from './ValidateForm';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
order the imports

The order of import should be as below:

react
redux
component file
utils ile
constants file
css file